src/overview.md

```diff
@@ -0,0 +1,126 @@
+---
+layout: page
+title: Overview
+---
+
+# Overview
+
+Workbox is a collection of libraries and build tools that make it easy to
+store your website's resources locally, on your users' devices. Consider Workbox
+if you want to:
+
+* Make your site work offline.
+* Improve load performance on repeat-visits. You can use Workbox to store
```

Comment:
I'll update the next section to include more of this nuance, but I'm going to leave this first section as is. The goal is to concisely explain the use cases / benefits of Workbox.